beagle/src/main/kotlin/br/com/zup/beagle/android/data/serializer/SerializerUtils.kt

```diff
@@ -16,8 +16,8 @@
 
 package br.com.zup.beagle.android.data.serializer
 
-import br.com.zup.beagle.annotation.RegisterAction
-import br.com.zup.beagle.annotation.RegisterWidget
+import br.com.zup.beagle.android.annotation.RegisterAction
+import br.com.zup.beagle.android.annotation.RegisterWidget
 import java.util.Locale
```

Comment:
<table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-android/blob/3b4731002a97ee566e2e282d363c4a805938ab39/beagle/src/main/kotlin/br/com/zup/beagle/android/data/serializer/SerializerUtils.kt#L21"></span>The import 'java.util.Locale' is unused.</td>
    </tr>
  </tbody>
</table>